contributor-guide/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,5 +1,82 @@
 = Release Notes
 :navtitle: Release Notes
 
+Writing release notes is crucial for informing users about the changes made in the latest version of a library. While the Boost community does not have a strict format for release notes (some libraries have a _change log_, others incorporate release information into their main documentation), there are however best practices and common structures followed by maintainers. 
 
-*Content under construction.*
\ No newline at end of file
+== Release Note Checklist
+
+Only include pertinent information in your release notes. Here is a checklist to go through to ensure all aspects are covered. There are obviously differences between a new library where the release notes might focus on a good <<_summary>>, <<_acknowledgements>> and <<_known_issues>>, and an update where the focus will be on <<_breaking_changes>>, <<_api_parameters>>, <<_new_features>> and <<_bug_fixes>>.
```

Comment:
This is also related to the point above. Internal release notes usually include everything. It's the boost release notes where we want to make these distinctions.

Reply:
Removed this paragraph, as the distinction between the two sets of release notes is covered elsewhere. 
FYI: @alandefreitas